drivers/i2s/i2s_cavs.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief I2S bus (SSP) driver for Intel CAVS.
+ *
+ * Limitations:
+ * - DMA is used in simple single block transfer mode (with linked list
+ *   enabled) and "interrupt on full transfer completion" mode.
+ */
+
+#ifndef _I2S_CAVS_H_
+#define _I2S_CAVS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct ssp {
```

Comment:
Three-letter struct names are likely to collide and don't belong in drivers.  Maybe "i2s_cavs_ssp" instead?

Reply:
sure. will make the change.